docs/cfp.md

```diff
@@ -7,66 +7,150 @@ hide:
 
 ### Program chairs:
 
-* Laurie Stephey, Lawrence Berkeley National Lab
 * Alberto Madona Madonna, CSCS
```

Comment:
The middle name is an intruder here and in the organizers page :eyes:

Also, I wonder if my affiliation (Swiss National Supercomputing Centre) should be spelled explicitly for consistency with the others.